src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -192,6 +192,16 @@ def _maybe_direct_owner(self, record: ClusterInfo) -> str | None:
         return record.creator
 
 
+class ClusterDetailsOwnership(Ownership[ClusterDetails]):
+    """Determine ownership of clusters in the workspace.
+
+    This is the cluster creator (if known).
+    """
+
+    def _maybe_direct_owner(self, record: ClusterDetails) -> str | None:
```

Comment:
convert from `ClusterDetails` to `ClusterInfo` and don't do any renames/new classes. we have "cluster" as a concept within our system, `ClusterDetails` and `ClusterInfo` are just the names of API generated classes which are slightly different views on the cluster entity.

TLDR: don't rename existing ownership classes.

Reply:
done